<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
<comment> Copyright (C) 2010 - 2014 by Pedro Mendes, Virginia Tech Intellectual </comment>
<comment> Properties, Inc., University of Heidelberg, and The University </comment>
<comment> of Manchester. </comment>
<comment> All rights reserved. </comment>

<comment> Copyright (C) 2009 by Pedro Mendes, Virginia Tech Intellectual </comment>
<comment> Properties, Inc., EML Research, gGmbH, University of Heidelberg, </comment>
<comment> and The University of Manchester. </comment>
<comment> All rights reserved. </comment>

 <comment> All rights reserved. </comment>
 <class>CQOptimizationResult</class>
 <widget class="CopasiWidget" name="CQOptimizationResult">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Optimization Result Window</string>
  </property>
  <property name="caption" stdset="0">
   <string>Fitting Result</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="mpLblResult">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Lucida Grande'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:16px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:x-large; font-weight:600;&quot;&gt;Optimization Result&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mpBtnUpdateModel">
       <property name="text">
        <string>Update Model</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mpBtnSave">
       <property name="text">
        <string>Save to File</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout">
     <item row="5" column="1">
      <widget class="QLineEdit" name="mpEditCPUTime">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="mpEditEvaluations">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLineEdit" name="mpEditSpeed">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="mpLblSpeed">
       <property name="text">
        <string>Evaluations/second [1/s]</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="mpLblEvaluations">
       <property name="text">
        <string>Function Evaluations</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="mpLblCPUTime">
       <property name="text">
        <string>CPU Time [s]</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="mpEditFailedEvaluationsExc">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="mpEditFailedEvaluationsNaN">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="mpLblFailedEvaluationsExc">
       <property name="text">
        <string>Failed Eval. (Exception)</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="mpLblFailedEvaluationsNaN">
       <property name="text">
        <string>Failed Eval. (NaN)</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="mpLblObjectiveValue">
       <property name="text">
        <string>Objective Value</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="mpEditObjectiveValue">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="mpTabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>210</width>
       <height>180</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QTableWidget" name="mpParameters">
      <attribute name="title">
       <string>Main</string>
      </attribute>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string>Parameter</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Lower Bound</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Start Value</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Value</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Upper Bound</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Gradient</string>
       </property>
      </column>
     </widget>
     <widget class="QWidget" name="mpLogPage">
      <attribute name="title">
       <string>Log</string>
      </attribute>
      <layout class="QGridLayout">
        <property name="margin">
          <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QPushButton" name="mpBtnFormatLog">
          <property name="text">
           <string>Get Formated Log</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
          <widget class="QWebView" name="mpLogWebView"/>
        </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CopasiWidget</class>
   <extends>QWidget</extends>
   <header>copasi/UI/copasiWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">copasi/UI/copasiWidget.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>mpBtnSave</sender>
   <signal>clicked()</signal>
   <receiver>CQOptimizationResult</receiver>
   <slot>slotSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpBtnUpdateModel</sender>
   <signal>clicked()</signal>
   <receiver>CQOptimizationResult</receiver>
   <slot>slotUpdateModel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpBtnFormatLog</sender>
   <signal>clicked()</signal>
   <receiver>CQOptimizationResult</receiver>
   <slot>slotFormatLog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
